WooCommerce/Classes/POS/Presentation/CartView.swift

```diff
@@ -16,9 +17,9 @@ struct CartView: View {
                 .font(.title)
                 .foregroundColor(Color.white)
             ScrollView {
-                ForEach(viewModel.productsInCart, id: \.product.productID) { cartProduct in
-                    ProductRowView(cartProduct: cartProduct) {
-                        viewModel.removeProductFromCart(cartProduct)
+                ForEach(viewModel.itemsInCart, id: \.item.productID) { cartItem in
```

Comment:
Good catch, and a nice reminder to get an eye on the console!

```
ForEach<Array<CartItem>, Int64, ModifiedContent<ModifiedContent<ItemRowView, _BackgroundStyleModifier<Color>>, _PaddingLayout>>: the ID 22 occurs multiple times within the collection, this will give undefined results!
```

Resolved here: https://github.com/woocommerce/woocommerce-ios/pull/12882/commits/9686125b628ecd2d9492e1197c66d24a1bc0c13b